dozer/cogs/moderation.py

```diff
@@ -47,6 +47,19 @@ async def check_audit(guild, event_time=None):
         except Forbidden:
             return None
 
+    @discord.ext.tasks.loop(hours=168)
+    async def nm_kick(self):
+        """Kicks people who have not done the new member process within a set amount of time."""
+        getLogger("dozer").debug("Starting nm_kick cycle...")
+        entries = await NewMemPurgeConfig.get_by()
+        for entry in entries:
+            guild = self.bot.get_guild(entry.guild_id)
```

Comment:
Should there be a check here to make sure the bot is still in that guild, otherwise it would fail if it reached a guild it was no longer in

Reply:
Yep, added